<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1068</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>321</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border-radius:10px; background-color: palette(base);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="placeholderText">
     <string>Raw Output</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="splitter">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>321</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border-radius:10px; background-color: palette(base);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="placeholderText">
     <string>Sliced Output</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>80</y>
      <width>231</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {border-radius: 10px; background-color: palette(base);}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Read all pages</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>45</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="pgcount">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLCDNumber{
  background-color: rgb(0, 0, 0);
  border: 2px solid rgb(113, 113, 113);
  border-width: 2px;
  border-radius: 5px;
  color: rgb(255, 255, 255);
}</string>
     </property>
    </widget>
   </widget>
   <widget class="QLCDNumber" name="counts">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>450</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">QLCDNumber{
  background-color: rgb(0, 0, 0);
  border: 2px solid rgb(113, 113, 113);
  border-width: 2px;
  border-radius: 5px;
  color: rgb(255, 255, 255);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>450</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Word Count :</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad"/>
   <addaction name="separator"/>
   <addaction name="actionRead"/>
   <addaction name="separator"/>
   <addaction name="actionDuplicates"/>
   <addaction name="separator"/>
   <addaction name="actionClear"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1068</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="qtr.qrc">
     <normaloff>:/icons/pdf2.png</normaloff>:/icons/pdf2.png</iconset>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="toolTip">
    <string>load a pdf file</string>
   </property>
  </action>
  <action name="actionRead">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/text.png</normaloff>
     <normalon>:/icons/text2.png</normalon>:/icons/text.png</iconset>
   </property>
   <property name="text">
    <string>Read</string>
   </property>
   <property name="toolTip">
    <string>Extract text</string>
   </property>
  </action>
  <action name="actionDuplicates">
   <property name="icon">
    <iconset resource="qtr.qrc">
     <normaloff>:/icons/remove.png</normaloff>:/icons/remove.png</iconset>
   </property>
   <property name="text">
    <string>Duplicates</string>
   </property>
   <property name="toolTip">
    <string>Remove Duplicates</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="qtr.qrc">
     <normaloff>:/icons/clear.png</normaloff>:/icons/clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>Reset and clear</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="qtr.qrc">
     <normaloff>:/icons/raw.png</normaloff>:/icons/raw.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionSave_Sliced_Output">
   <property name="icon">
    <iconset resource="qtr.qrc">
     <normaloff>:/icons/saveas.png</normaloff>:/icons/saveas.png</iconset>
   </property>
   <property name="text">
    <string>Save Sliced Output</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="qtr.qrc"/>
 </resources>
 <connections/>
</ui>
